content/docs/start/bitbucket.md

```diff
@@ -38,28 +35,27 @@ all the supported CI systems.
              - python train.py
 
              - cat metrics.txt > report.md
-             - cml-publish plot.png --md >> report.md
-             - cml-send-comment report.md
+             - cml publish plot.png --md >> report.md
+             - cml send-comment report.md
    ```
 
-5. In your text editor of choice, edit line 15 of `train.py` to `depth = 12`.
+4. In your text editor of choice, edit line 16 of `train.py` to `depth = 5`.
```

Comment:
I assume the code is the same as in the GitHub example?

Reply:
conversation moved to https://github.com/iterative/cml.dev/pull/267#discussion_r908438665